core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1777,7 +1787,7 @@ class KafkaApis(val requestChannel: RequestChannel,
           s"client ${request.header.clientId}.")
         responseBody
       }
-      sendResponseMaybeThrottle(request, createResponse)
+      sendResponseMaybeThrottle(controllerMutationQuota, request, createResponse, None)
```

Comment:
nit: Make the last argument a default arg or use named argument since it is not clear what None is.